index.html

```diff
@@ -9,6 +9,12 @@
   <link rel="stylesheet"  type="text/css" href="application.css">
 </head>
 <body>
-  <h1 class="title">Hello world</h1> 
+  <h1 class="title">Front-End Tools</h1> 
+  <div class="tech-card">
+    <img class="tech-card-image" src="assets/react_icon.svg"></img>
```

Comment:
`img` tags are self closing: `<img class="tech-card-image" src="assets/react_icon.svg" />` you can't put any children inside of it